### PR TITLE
doc(support-bundle): add feature doc

### PR DESCRIPTION
**Pull request by c3y1huang**
https://github.com/longhorn/longhorn/issues/2759

### CODE REVIEW

File: content/docs/1.4.0/references/settings.md [hunk 1]
```diff
@@ -264,6 +266,20 @@ This setting allows Longhorn to automatically delete the `orphan` resource and i
 
 The value in seconds specifies the timeout of the engine to the replica(s), and the value should be between 8 to 30 seconds. The default value is 8 seconds.
 
+#### Support Bundle Manager Image
+
+Longhorn uses the support bundle manager image to generate the support bundles.
```
Comment (innobead) [inline content/docs/1.4.0/references/settings.md:271]: Please have an example value for this, and explain there will be default one during instal or upgrade.

Comment (c3y1huang) [reply content/docs/1.4.0/references/settings.md:271]: Updated.

File: content/docs/1.4.0/references/settings.md [hunk 2]
```diff
@@ -264,6 +266,25 @@ This setting allows Longhorn to automatically delete the `orphan` resource and i
...
+
+There will be a default image given during installation and upgrade. You can also change it in the settings.
+
+An example of the support bundle manager image:
+> Default: `longhornio/support-bundle-kit:v0.0.13`
```
Comment (innobead) [inline content/docs/1.4.0/references/settings.md:276]: `0.0.14`

File: content/docs/1.4.0/references/settings.md [hunk 3]
```diff
@@ -264,6 +266,25 @@ This setting allows Longhorn to automatically delete the `orphan` resource and i
...
+
+#### SupportBundle Failed History Limit
```
Comment (innobead) [inline content/docs/1.4.0/references/settings.md:278]: `Support Bundle Failed History Limit`

**Review by innobead — CHANGES_REQUESTED** [review]
In general, LGTM. Just a few comments needed to resolve.

File: content/docs/1.4.0/references/settings.md (hunk 2 above)
Comment (c3y1huang) [reply content/docs/1.4.0/references/settings.md]: Updated.

File: content/docs/1.4.0/references/settings.md (hunk 3 above)
Comment (c3y1huang) [reply content/docs/1.4.0/references/settings.md]: Fixed.

File: content/docs/1.4.0/references/settings.md [hunk 4]
```diff
@@ -32,6 +32,8 @@ weight: 1
   - [Backing Image Recovery Wait Interval](#backing-image-recovery-wait-interval)
   - [Orphaned Data Automatic Deletion](#orphaned-data-automatic-deletion)
   - [Engine to Replica Timeout](#engine-to-replica-timeout)
+  - [Support Bundle Manager Image](#support-bundle-manager-image)
+  - [SupportBundle Failed History Limit](#support-bundle-failed-history-limit)
```
Comment (innobead) [inline content/docs/1.4.0/references/settings.md:36]: `Support Bundle Failed History Limit`

**Review by innobead — CHANGES_REQUESTED** [review]

File: content/docs/1.4.0/references/settings.md (hunk 4 above)
Comment (c3y1huang) [reply content/docs/1.4.0/references/settings.md]: Fixed.

**Review by innobead — APPROVED** [review]

**Pull request merged**